examples/gnrc_minimal/Makefile

```diff
@@ -13,7 +13,7 @@ BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h
 #USEMODULE += xbee
 
 ## set default UART to use in case none was defined
-#XBEE_UART ?= "UART_NUMOF-1"
+#XBEE_UART ?= "1"
```

Comment:
A bit, it should be avoided somehow, there is no warning or similar thing showing a configuration inconsistency... I'll try to add it somewhere...